lib/elixir_sense/core/state.ex

```diff
@@ -50,7 +50,7 @@ defmodule ElixirSense.Core.State do
     current_module     = get_current_module(state)
     current_imports    = state.imports    |> :lists.reverse |> List.flatten
     current_requires   = state.requires   |> :lists.reverse |> List.flatten
-    current_aliases    = state.aliases    |> :lists.reverse |> List.flatten
+    current_aliases    = state.aliases    |> List.flatten |> Enum.uniq_by(& elem(&1, 0)) |> Enum.reverse
```

Comment:
Vaguely alarmed the formatter hasn't been run on this project, but that's well outside the scope of your changes.

Reply:
@axelson I don't have anything big in the works. Feel free to run formatter